server/@types/shared/models/Cas2Application.ts

```diff
@@ -19,5 +19,9 @@ export type Cas2Application = (Application & {
     telephoneNumber?: string;
     assessment?: Cas2Assessment;
     timelineEvents?: Array<Cas2TimelineEvent>;
+    allocatedPomName?: string;
+    currentPrisonName?: string;
+    allocatedPomEmailAddress?: string;
+    assignmentDate?: string;
```

Comment:
Shouldn't be needed 

https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-ui/pull/900/files#diff-6d5eefe3d87e4157ea22c1501c556091c19b41dc4a2b071cec4716e4fac8b100R22-R25


Could you remove all the `@types/shared/models` changes as there is a separate process to pull in new/amended types from the API